### ISSUE TITLE
Doesn´t work on iPhone 

### ISSUE THREAD

**Issue by channelsurfer** [forrestkirby/flipcard-element#3]
The Flipcard element on the iPhone shows a wired behavior. It turns and you see the back of the cards as soon it sets the front is just mirrored. Check the screenshot for info. Any hints?

**Comment by forrestkirby** [comment]
Could you try replacing [line 17](https://github.com/forrestkirby/flipcard-element/blob/master/hd-flipcard/css/flipcard.css#L17) of the CSS

```
	-moz-backface-visibility: hidden;
```

by

```
	-webkit-backface-visibility: hidden;
```

?

**Comment by channelsurfer** [comment]
great! this works now... thank you... 

is there also a way to center the back-content of the card vertically in the middle?

**Comment by forrestkirby** [comment]
fixed with [commit 6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36 ](https://github.com/forrestkirby/flipcard-element/commit/6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36)

---

> is there also a way to center the back-content of the card vertically in the middle?

please open a new issue for this question. thanks

**Comment by forrestkirby** [comment]
fixed with [commit 6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36 ](https://github.com/forrestkirby/flipcard-element/commit/6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36)

---

> is there also a way to center the back-content of the card vertically in the middle?

please open a new issue for this question. thanks

**Comment by forrestkirby** [comment]
fixed with [commit 6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36 ](https://github.com/forrestkirby/flipcard-element/commit/6598ad7fce7a4bd5962ecccfaf0a46d93e3bcb36)

---

> is there also a way to center the back-content of the card vertically in the middle?

please open a new issue for this question. thanks
